static/app/components/charts/useChartZoom.tsx

```diff
@@ -78,51 +62,19 @@ interface Props {
  * the props that would be passed to the `BaseChart` as zoomRenderProps.
  */
 export function useChartZoom({
-  period,
-  start,
-  end,
-  utc,
   router,
   onZoom,
   usePageDate,
   saveOnZoom,
-  onChartReady,
-  onRestore,
-  onDataZoom,
-  onFinished,
   xAxisIndex,
   showSlider,
   chartZoomOptions,
-  xAxis,
-  disabled,
-}: Omit<Props, 'children'> = {}) {
-  const [currentPeriod, setCurrentPeriod] = useState<Period | undefined>({
-    period: period!,
-    start: getDate(start),
-    end: getDate(end),
-  });
-  const [history, setHistory] = useState<Period[]>([]);
-
-  const [zooming, setZooming] = useState<(() => void) | null>(null);
```

Comment:
using state here throws warnings and they're not used to rerender only for storing internal state.